### ISSUE TITLE
Installer Requirements

### ISSUE THREAD

**Issue by Skavenger0** [brave/browser-laptop#417]
I'm testing Brave at home and at work aswel, looks like a fantastic idea and I am one of our SysAdmins. (Yes I know its not ready yet but I'm just curious at this point)

Issue:

1: We (like many companies) block executable running from Program Data or the Users Profile.
2: The installation by default is for a single user rather than multiple people using the same computer.

Request:
Ability to choose install location for all users that use the computer.
This means companies and administrators can securely install the software without the need to amend software restriction policies and will be more willing to adopt the browser at release time.

Basically if you want to be able to target corporate networks aswel as home users then there will need to be an alternate installer at final release.

**Comment by bbondy** [comment]
We have an MSI which should work similar to:
https://get.slack.help/hc/en-us/articles/212475728-Deploying-Slack-to-your-organization-via-MSI

If we sent it to you could you verify that it does?

**Comment by Skavenger0** [comment]
Brilliant, I will give it a go and let you know how I get on.

**Comment by bbondy** [comment]
Try from here using the Slack instructions above though.
https://github.com/brave/browser-laptop/releases/download/v0.7.10dev/Setup.msi

**Comment by luixxiul** [comment]
Closing for the moment.

**Comment by thesohelshaikh** [comment]
https://github.com/brave/browser-laptop/releases
